### ISSUE TITLE
Error importing CanvasTemplate_pyside2

### ISSUE THREAD

**Issue by TobiasNils** [pyqtgraph/pyqtgraph#2320]
### Short description
Importing `CanvasTemplate_pyside2` fails with `ModuleNotFoundError`

### Code to reproduce

```python
from pyqtgraph.canvas import CanvasTemplate_pyside2
```

### Expected behavior
Template should be imported

### Real behavior
`ModuleNotFoundError` is raised
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib64/python3.9/site-packages/pyqtgraph/__init__.py", line 247, in <module>
    from .canvas import *
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib64/python3.9/site-packages/pyqtgraph/canvas/__init__.py", line 1, in <module>
    from .Canvas import *
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib64/python3.9/site-packages/pyqtgraph/canvas/Canvas.py", line 10, in <module>
    ui_template = importlib.import_module(
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib64/python3.9/site-packages/pyqtgraph/canvas/CanvasTemplate_pyside2.py", line 85, in <module>
    from CanvasManager import CanvasCombo
  File "/home/tna/WorkSpace/Devices/HPC/gui/.venv/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'CanvasManager'
```

### Tested environment(s)

 * PyQtGraph version: 0.12.4
 * Qt Python binding: PySide2 5.15.2.1 Qt 5.15.2
 * Python version: 3.9
 * NumPy version: 1.22.4
 * Operating system: Fedora 35
 * Installation method: pip

### Additional context
Error turned up first in the context of compiling my app with pyinstaller. 
In the app I was not loading the template eplicitly. 

### Possible Solution
Can be solved in my case by making the import of `CanvasManager` in line `85` of [CanvasTemplate_pyside2.py](https://github.com/pyqtgraph/pyqtgraph/blob/master/pyqtgraph/canvas/CanvasTemplate_pyside2.py) relative, as in the templates for other Qt versions, i.e.
~~`from CanvasManager import CanvasCombo`~~
`from .CanvasManager import CanvasCombo`

**Comment by pijyoi** [comment]
In the same vein, the following should also be refreshed.
```
$ git grep "PyQt5 UI code generator 5\.[0-9]\."
canvas/CanvasTemplate_pyqt5.py:# Created by: PyQt5 UI code generator 5.7.1
canvas/TransformGuiTemplate_pyqt5.py:# Created by: PyQt5 UI code generator 5.5.1
console/template_pyqt5.py:# Created by: PyQt5 UI code generator 5.5.1
flowchart/FlowchartCtrlTemplate_pyqt5.py:#      by: PyQt5 UI code generator 5.0.1
flowchart/FlowchartTemplate_pyqt5.py:#      by: PyQt5 UI code generator 5.0.1
graphicsItems/ViewBox/axisCtrlTemplate_pyqt5.py:#      by: PyQt5 UI code generator 5.0.1
graphicsItems/ViewBox/axisCtrlTemplate_pyside2.py:#      by: PyQt5 UI code generator 5.0.1
imageview/ImageViewTemplate_pyqt5.py:#      by: PyQt5 UI code generator 5.0.1
```

**Comment by pijyoi** [comment]
This issue should have been fixed by #2448

**Comment by j9ac9k** [comment]
pijyoi beat me to it; anyway closing out this issue as all the template files will be gone in the next release.
